client/src/app/util/jwt.interceptor.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { HttpRequest, HttpHandler, HttpEvent, HttpInterceptor } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { AuthService } from '../service/auth.service';
+
+
+@Injectable()
+export class JwtInterceptor implements HttpInterceptor {
+    constructor(private authService: AuthService) { }
+
+    intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+        let currentUser = this.authService.currentUserValue;
+        if (currentUser && currentUser.token) {
+            request = request.clone({
+                setHeaders: {
+                    Authorization: `Bearer ${currentUser.token}`
```

Comment:
As mentioned earlier, token here is entire object 
```json
{
	"accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6Ik...",
	"expiresAt": 1800
}
```
so authorization will not be successful.